### PR TITLE
#161290904 Authenticated Users Rate Article

### PR DESCRIPTION
**Pull request by ajimae**
#### What does this PR do?

- Allow authenticated users to rate a published article on a scale of 1 to 5

#### Description of Task to be completed?

- Add a helper file to hold all functions
- Write code to handle all perceivable errors
- Update article route and controller files to allow rating functionality
- Update article table to accommodate ratings
- Write a test to validate functionality

#### How should this be manually tested?

- After cloning this repo, navigate into the clone repo and `run npm install` to install all the dependencies and dev-dependencies
- Using the `.env.sample` content, create a `.env` in your root directory and fill in the fields
- To run on the development mode, run `npm run dev`
- Navigate to `localhost:3000/api/v1/articles/ratings/:articleId`

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
[#161290904](https://www.pivotaltracker.com/story/show/161290904)

#### Screenshots (if appropriate)

_For a successful rating_
image

_For a second rating from same user_
image

_For unknown article ID_
image

#### Questions:
N/A

### CODE REVIEW

**Review by desaawa — APPROVED** [review]
Resolve merge conflicts and merge

File: server/src/test/integration tests/tagsTest.js [hunk 1]
```diff
@@ -83,21 +83,21 @@ describe('Tags Test Initializations', () => {
...
+    //       tags: 'bars,foos,philosophical,smart,brainiac',
+    //     })
+    //     .end((error, response) => {
+    //       if (error) done(error);
+    //       expect(response.status).to.equal(200);
+    //       expect(response.body).to.be.an('object');
+    //       expect(response.body.message).to.equal('tags updated successfully');
+    //       done();
+    //     });
+    // });
```
Comment (andela-moe) [inline server/src/test/integration tests/tagsTest.js]: Any reason why this snippet of code is commented? If it is not in use why not take it out?

**Review by andela-moe — COMMENTED** [review]
Please review the comment on this PR

**Review by ovieokeh — CHANGES_REQUESTED** [review]
Coveralls coverage has plummeted drastically. Please look into it.

Fixed

**Pull request merged**